include/itkLabelSetUtils.h

```diff
@@ -477,8 +477,9 @@ doOneDimensionErode(TInIter &          inputIterator,
   {
     iscale = image_scale;
   }
-  const RealType  magnitude = (m_MagnitudeSign * iscale * iscale) / (2.0 * Sigma);
-  LineBufferType  LineBuf(LineLength);
+  const RealType magnitude = (m_MagnitudeSign * iscale * iscale) / (2.0 * Sigma);
+  LineBufferType LineBuf(LineLength);
+  LineBuf.Fill(0.0);
```

Comment:
`LineBufferType` is an alias of `itk::Array`: https://github.com/InsightSoftwareConsortium/ITKLabelErodeDilate/blob/f427170b4432a71d83bc2ce90ccd881b0e979996/include/itkLabelSetUtils.h#L251

So instead of calling `LineBuf.Fill(0.0)`, you might also initialize its elements directly by the `Array(SizeValueType dimension, const ValueType & value)` constructor:

```cpp
    LineBufferType LineBuf(LineLength, 0.0);
```

No show-stopper, just a suggestion.

Reply:
This is what I had tries locally, but wasn't able to verify with valgrind on Friday.

Reply:
@N-Dekker adopted `LineBufferType LineBuf(LineLength, 0.0);` in  0db0238.

@blowekamp Let us know whenever you can afford trying with valgrind.